README.md

```diff
@@ -1,102 +1,199 @@
 
-There are numerous combinations of source clusters, target clusters, and platforms. While the tools provided in this repository might work with various combinations, they might not support breaking changes between different source and target versions. Below is a list of supported source and target versions and platforms.
+- **User-Friendly Interface via [Migration Console](https://github.com/opensearch-project/opensearch-migrations/blob/main/docs/migration-console.md)**: Command Line Interface (CLI) that guides you through each migration step.
 
-### Supported Source and Target Versions
-* Elasticsearch 6.x (Coming soon...)
-* Elasticsearch 7.0 - 7.17.x
-* OpenSearch 1.x
-* OpenSearch 2.x
+- **Flexible Deployment Options**:
+  - **[AWS Deployment](https://aws.amazon.com/solutions/implementations/migration-assistant-for-amazon-opensearch-service/)**: Fully automated deployment to AWS.
+  - **[Local Docker Deployment](/TrafficCapture/dockerSolution/README.md)**: Run the solution locally in a container for testing and development.
 
-### Supported Source and Target Platforms
-* Self-managed (hosted by cloud provider)
-* Self-managed (on-premises)
-* Managed cloud offerings (e.g., Amazon OpenSearch, Amazon OpenSearch Serverless)
+## Supported Versions and Platforms
 
-## Build and deploy
+- **Tested Migration Paths**:
+  - Elasticsearch 6.8 to OpenSearch 1.3, 2.14
+  - Elasticsearch 7.10.2 to OpenSearch 1.3, 2.14
+  - Elasticsearch 7.17 to OpenSearch 1.3, 2.14
+  - OpenSearch 1.3 to OpenSearch 2.14
 
-### Local deployment
+Note that testing is done on specific minor versions, but any minor versions within a listed major version are expected to work.
 
-A containerized end-to-end solution can be deployed locally using the 
-[Docker Solution](TrafficCapture/dockerSolution/README.md).
+- **Platforms**:
+  - Self-managed (cloud provider hosted)
+  - Self-managed (on-premises)
+  - Managed cloud offerings (e.g., Amazon OpenSearch, Amazon OpenSearch Serverless)
 
-### AWS deployment
+While untested, alternative cloud providers are expected to work.
 
-Refer to [AWS Deployment](deployment/README.md) to deploy this solution to AWS.
+## Issue Tracking
 
-## Developer contributions
+We encourage users to open bugs and feature requests in this GitHub repository. 
 
-### Code Style
+**Encountering a compatibility issue or missing feature?**
+
+- [Search existing issues](https://github.com/opensearch-project/opensearch-migrations/issues) to see if it’s already reported. If it is, feel free to **upvote** and **comment**.
+- Can’t find it? [Create a new issue](https://github.com/opensearch-project/opensearch-migrations/issues/new/choose) to let us know.
+
+For issue prioritization and management, the migrations team uses Jira, but uses GitHub issues for community intake:
+
+https://opensearch.atlassian.net/
+
+## Project Structure
```

Comment:
From this second down seems like it belongs in a DEVELOPER_GUIDE.md, rather than in main readme, what do you think?

Reply:
We have no DEVELOPER_GUIDE.md at the root convention but that's a good convention. I'll add one.

Reply:
Added